jsonrpc/server_test.go

```diff
@@ -475,6 +482,10 @@ func TestHandle(t *testing.T) {
 			res:              `{"jsonrpc":"2.0","error":{"code":-32603,"message":"Internal error"},"id":1}`,
 			checkFailedEvent: true,
 		},
+		"empty optional param": {
+			req: `{"jsonrpc": "2.0", "method": "emptyOptionalParam", "params": {}, "id": 1}`,
+			res: `{"jsonrpc":"2.0","result":0,"id":1}`,
+		},
```

Comment:
Sorry to be pedantic. I thought we wanted to check for the following case:

`req: {"jsonrpc": "2.0", "method": "emptyOptionalParam", "id": 1},`, right?

It would be good to add this too.